README.md

```diff
@@ -21,7 +21,8 @@ Because the package installation is async, it returns a promise that resolves wh
 
 module.exports = {
   activate() {
-    require('atom-package-deps').install()
+    # replace the example argument 'linter-ruby' with the name of this linter package
```

Comment:
Coffeescript, Javascript, ... all the same :wink:

Fixed it in 28c92c00d16cdf095dcc5fc0dda510d5ff146545